ravindu14/SliitSpmProject
# Feature task id #104

how to test
1. move to 'http://localhost:4200/tables' url
2.fill all the feilds
3. user should be able to save the data to the database
